packages/opentelemetry-plugin-http/test/functionals/http-enable.test.ts

```diff
@@ -751,6 +751,14 @@ describe('HttpPlugin', () => {
           SpanKind.CLIENT
         );
       });
+
+      it('should not set span as active in context for outgoing request', async () => {
+        assert.deepStrictEqual(getActiveSpan(context.active()), undefined);
+        await httpRequest.get(`${protocol}://${hostname}:${serverPort}/test`);
```

Comment:
I think the fix in the PR is correct, but this test is not actually testing the logic in the PR.
If you run it on master, it will pass as well, probably due to the promise-await semantics.

I suggest a straight forward test on the request callback:

```js
      it('should not set span as active in context for outgoing request', (done) => {
        http.get(`${protocol}://${hostname}:${serverPort}/test`, (res) => {
          assert.deepStrictEqual(getActiveSpan(context.active()), undefined);
          done();
        });
      });
```

Reply:
You are right, i will refactor that test